### ISSUE TITLE
How can I use this with Nuxt.js?

### ISSUE THREAD

**Issue by SHxKM** [gu-fan/vue-edge-check#2]
Thanks for the plugin. Can I use this with Nuxt.js?

I've added this to my `/plugins` folder?

```js
// vue-edge-check.js
import Vue from 'vue'
import EdgeCheck from 'vue-edge-check'
Vue.use(EdgeCheck)
```

And this in my `nuxt.config.js`:

```js
plugins: [
    ...
    { src: '@/plugins/vue-edge-check.js', ssr: false }
  ],
```

However It's still not taking effect and Vue is complaining about:

> Property or method "$isEdgeRight" is not defined on the instance but referenced during render.

When I use it on my component like this:

```vue
<div class="flex flex-col ..." :class="{'is-edge-back':$isEdgeLeft,'is-edge-forward':$isEdgeRight}">
```

What am I missing?
